Makefile

```diff
@@ -156,3 +158,11 @@ swagger-gen:
 		--entrypoint hack/generate-swagger-api.sh \
 		-e GOPATH=/go \
 		quay.io/goswagger/swagger:0.7.4
+
+.PHONY: swagger-docs ## preview the API documentation
```

Comment:
oh! looks like it has to be on the line below, `make help` doesn't seem to pick it up if it's here :cry:

we can fix it in a follow up if needed, no blocker